### ISSUE TITLE
Bookmarks > Right Click Folder > Open All In Tabs goes to Default instead of current

### ISSUE THREAD

**Issue by ScottRFrost** [jonathanKingston/sea-containers#46]
If I'm looking at a tab in container "Personal" and click Open All In Tabs on a folder from my bookmarks, I'd like all of those tabs to load in the current container, not into the default container.

**Comment by tradewatcher** [comment]
Yes i have the same issue but i can prevent it with opening a new tab in the selected container, visit a website and then click "Open All in Tabs".
